File: languages/GBL/Java/Sample.java
// BEGIN ...
package org.softlang.gbl;
import org.softlang.gbl.Syntax.*;
// END ...
public class Sample {
    public World smallWorld() {
        World w = new World();
        Person p1 = new Person();
        Person p2 = new Person();
        p1.name = "joe";
        p1.buddy = p2;
        p2.name = "bill";
        p2.buddy = p1;
        return w;
    }
}

File: languages/BL/Haskell/Language/GBL/V2/Sample.hs
-- BEGIN ...
module Language.GBL.V2.Sample where
import Language.GBL.V2.Syntax
-- END ...
smallWorld :: World
smallWorld = World [p1, p2]
  where
    p1, p2 :: Person
    p1 = Person { getName = "joe", getBuddy = Just p2 }
    p2 = Person { getName = "bill", getBuddy = Just p1 }
